nodes/core/core/25-catch.html

```diff
@@ -85,6 +85,17 @@
 #node-input-catch-target-container li:hover .node-input-target-node-sublabel {
     background: #f0f0f0;
 }
+.node-input-target-sort-type-div {
```

Comment:
Need to be very careful to ensure this css class name is scoped to just the catch node.

Note how the previous rule is scoped to #node-input-catch-target-container (for example).

This comment applies to _all_ node-specific css changes.